etna/transforms/base.py

```diff
@@ -127,6 +133,19 @@ def transform(self, ts: TSDataset) -> TSDataset:
         ts = self._update_dataset(ts=ts, columns_before=columns_before, df_transformed=df_transformed)
         return ts
 
+    def _save_exog_last_date(self, df_exog: pd.DataFrame):
```

Comment:
We should probably discuss is this solution ok or not. Are there any alternatives? May be doing it inside of `BaseTransform` isn't a good idea.

Reply:
Alternativly, we can override `fit` method in `ExogShiftTransform` and put this logic there.